homeassistant/components/blebox/button.py

```diff
@@ -15,20 +17,23 @@ async def async_setup_entry(
     async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Set up a BleBox button entry."""
-    create_blebox_entities(
-        hass, config_entry, async_add_entities, BleBoxButtonEntity, "buttons"
-    )
+    entities: list[BleBoxButtonEntity] = []
+
+    for feature in get_blebox_features(hass, config_entry, "buttons"):
+        entities.append(BleBoxButtonEntity(feature))
```

Comment:
`entities` could be a list comprehension. Same for the other platforms.

Reply:
Implemented - but moved to each platform